packages/components/src/components/legends/as-legend-size-bins-point/as-legend-size-bins-point.tsx

```diff
@@ -44,26 +44,31 @@ export class LegendSizeBinsPoint {
       ? FAKE_BORDER_SIZE * 2
       : 0;
 
-    const size = `${Math.round(data.width) + sizeOffset}px`;
+    const SIZE = Math.round(data.width) + sizeOffset;
+    const SIZE_PX = `${SIZE}px`;
+    const MAX_SIZE = this.maxSize;
+    const MAX_SIZE_PX = `${MAX_SIZE}px`;
 
     const mask = this.getMask(data);
 
-    const style: any = {
-      backgroundColor: data.color,
-      border: strokeStyle,
-      height: size,
-      width: size,
-      ...mask
-    };
-
     const wrapperStyle: any = { };
 
     if (this.orientation === 'horizontal') {
-      wrapperStyle.height = `${this.maxSize}px`;
+      wrapperStyle.height = MAX_SIZE_PX;
     } else if (this.orientation === 'vertical') {
-      wrapperStyle.width = `${this.maxSize}px`;
+      wrapperStyle.width = MAX_SIZE_PX;
     }
 
+    const STYLE_SIZE_PX = SIZE > MAX_SIZE ? MAX_SIZE_PX : SIZE_PX;
```

Comment:
So, if you do this you're making the point smaller than it needs to be. The 2 pixels are being added to account for the border and box-sizing: border-box.

I think what we should do is make the wrapper Math.max(SIZE, MAX_SIZE).

Reply:
I see! You're right, let's try that